src/Core/AuthenticationHelpers/AppServiceAuthentication.cs

```diff
@@ -103,10 +103,11 @@ public class AppServiceClaim
                     // Logging the parsing failure exception to the console, but not rethrowing
                     // nor creating a DataApiBuilder exception because the authentication handler
                     // will create and send a 401 unauthorized response to the client.
-                    logger.LogError($"Failure processing the AppService EasyAuth header.\n" +
-                        $"{HttpContextExtensions.GetLoggerCorrelationId(context)}" +
-                        $"{error.Message}\n" +
-                        $"{error.StackTrace}");
```

Comment:
no need of stacktrace?

Reply:
No, the exception (and associated stacktrace) will be logged/tracked automatically by supplying the exception object to the `exception` param:  `LogError(exception: ex, message:"msg")`